src/components/assistant-code-block.tsx

```diff
@@ -67,6 +75,27 @@ const LanguageBlock = ( props: ContextProps & CodeBlockProps ) => {
 					className="h-auto mr-2 !px-2.5 py-0.5 !p-[6px] font-sans select-none"
 					iconSize={ 16 }
 				></CopyTextButton>
+				{ 'language-sh' === props.className && (
+					<CopyTextButton
+						icon={ preformatted }
+						text={ content }
+						label={ __( 'Copy and open terminal' ) }
```

Comment:
The design says: 'Open in terminal' but I preferred to say copy and open terminal.

Another alternative is sending a "notification" to inform the user the command has being copied.

Reply:
`Copy and open terminal` feels awkwardly long in comparison to the other buttons.

I like the idea of using a notification to inform the user that we’ve copied the command. If we implement that, we should do the same for the `Copy` button for consistency.

Reply:
Same here, the `Copy and open terminal` looks too long compared to the other buttons there. Plus, in other languages will be even longer (e.g. in Spanish this will be `Copiar y abrir el terminal`. It would be great if it could be a one/two words text.

Reply:
I presume adding the word `Copy` is because we want to convey that the command is copied to the clipboard. In this regard, we could inform about this in the terminal by printing a text (e.g. using `echo`).